### DISCUSSION TITLE
Linux Debian apt version so far behind Flatpak

### DISCUSSION THREAD

**Discussion by samuelawachie** [smplayer-dev/smplayer#995] [General]
I'm on Zorin OS (17.1) and the latest version of the app when it's installed from package manager repo is 21.10.0. This is miles behind the flatpak version available for me distro which is 24.5.0. And is also miles behind the latest version shown on the website. 
1. Is there a reason this apt version is so outdated? 
2. If I build and make from the binary using the tar (which is the latest version), is there a way I can test that as a portable install to see how it functions and looks? Will it even work? How can I transfer my Preferences if I  should like it? 
3. Has there been any major updates from version 21 to 24? In other words, what will I gain from updating, coz my current version serves well but it's just old now...
4. How would you compare the flatpak vs the Debian apt version if performance, stability/reliability and bug-free are the topmost considerations? I'm not looking for "security" if that comes at the expense of stuttering, or Sound issues.

**Comment by smplayer-dev** [comment]
Which version of Debian is that distribution based on?
You can get deb packages for several versions of debian here:
https://software.opensuse.org/download.html?project=home%3Asmplayerdev&package=smplayer
